frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerThread.java

```diff
@@ -283,15 +285,15 @@ public void run() {
         } catch (OutOfMemoryError oom) {
             logger.error("Out of memory error when creating workers", oom);
             status = HttpURLConnection.HTTP_ENTITY_TOO_LARGE;
-            if (java.lang.System.getenv("SM_TELEMETRY_LOG") != null) {
+            if (java.lang.System.getenv("SM_TELEMETRY_LOG_REV_2022_12") != null) {
```

Comment:
Why need get and check this env var everywhere? It only needs once and simply flag can be applied.

Reply:
Fixed